expr/ztests/ip-len.yaml

```diff
@@ -0,0 +1,11 @@
+zql: cut len(addr)
+
+input: |
+  #0:record[addr:ip]
```

Comment:
I hIghly recommend adding test coverage for net as well.

Reply:
Good idea.  The exposed an old bug with ipv6 network values.